build.gradle

```diff
@@ -44,6 +44,9 @@ signing {
     sign configurations.archives
 }
 
+def sonatypeUsername = System.getenv('sonatypeUsername')
```

Comment:
Thanks for the PR! Quick question, how does adding this work? Are we just changing it from using an environment variable instead of a gradle property?